scripts/generate-proto

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# Generate the go proto files and put them in the third_party/protobuf/bazel/ directory.
```

Comment:
this is a surprising shape to me. Why not just have a Bazel rule in the package where the .proto files are, which writes the generated files to the source tree?